subworkflows/local/utils_nfcore_tomte_pipeline/main.nf

```diff
@@ -221,8 +221,21 @@ def toolCitationText() {
     // Uncomment function in methodsDescriptionText to render in MultiQC report
     def citation_text = [
             "Tools used in the workflow included:",
+            "BCFtools (Danecek et al. 2021),",
+            "DROP (Yépez et al. 2021),",
+            params.switch_vep ? "EnsemblVEP (McLaren et al. 2016)," : "",
+            "fastp (Chen et al. 2018),",
             "FastQC (Andrews 2010),",
-            "MultiQC (Ewels et al. 2016)",
+            params.switch_drop_as ? "FRASER (Mertes et al 2021)," : "",
+            "GATK (McKenna et al. 2010),",
+            params.switch_stringtie ? "GFFCompare (Pertea et al. 2020)," : "",
+            "MultiQC (Ewels et al. 2016),",
```

Comment:
Stringtie is there also on line 237, is it like this so that  we get GFFCompare and Stringtie both in?

Reply:
If we don't run stringtie in tomte, then gffcompare won't be executed as it is dependent on the output from the stringtie process. This is to prevent gffcompare to pop up even if it wasn't executed

Reply:
Yes, but wouldn't it be better to put both together (if its allowed)?
`params.switch_stringtie ? "GFFCompare (Pertea et al. 2020),Stringtie..." : "","MultiQC (Ewels et al. 2016),",`